reference/concepts/boolean_logic.md

```diff
@@ -3,3 +3,11 @@
 We expect students to know [booleans](../types/boolean.md) (`TRUE` and `FALSE`) as a concept, and be familiar with logical `AND`, `OR`, `NOT`.
 
 Track specific exercises could cover precedence or unusual rules around boolean logic, but should not try to teach it from scratch.
+
+#### Implementations
```

Comment:
Could you also add the parent `## Exercises` section and within that add an entry for your exercise (see [this example](https://github.com/exercism/v3/blob/master/reference/types/boolean.md#exercises)).

Reply:
It might be useful to also add this information to the `reference/types/boolean.md` reference document: https://github.com/exercism/v3/blob/master/reference/types/boolean.md#exercises